zephyr/prj.conf

```diff
@@ -15,3 +15,27 @@ CONFIG_MBEDTLS_HEAP_SIZE=16384
 
 #CONFIG_BT_HCI_TX_STACK_SIZE=1024
 CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+
+CONFIG_FLASH=y
+CONFIG_FLASH_PAGE_LAYOUT=y
+
+CONFIG_NVS=y
+CONFIG_NVS_LOG_LEVEL_DBG=y
+CONFIG_MPU_ALLOW_FLASH_WRITE=y
+
+
+# CONFIG_DISK_ACCESS=y
+CONFIG_MAIN_STACK_SIZE=2048
+
+# Let __ASSERT do its job
+CONFIG_DEBUG=y
+
+CONFIG_LOG=y
+
+CONFIG_FLASH=y
+CONFIG_FLASH_MAP=y
+CONFIG_FLASH_PAGE_LAYOUT=y
+
+# Need this when storage is on MX25R64
+CONFIG_NORDIC_QSPI_NOR=y
+CONFIG_NORDIC_QSPI_NOR_FLASH_LAYOUT_PAGE_SIZE=4096
```

Comment:
If this is only needed for MX25R64, please consider to move this to the PIO.ini file or a separate board conf file.